src/main/java/org/kurdyukov/sensu/ZabbixServerOutput.java

```diff
@@ -51,7 +64,99 @@ public boolean isRunning() {
 
     @Override
     public void write(Message message) throws Exception {
-        // TODO: implement writing
+
+        LOG.info("connecting to zabbix server...");
+        Socket socket = new Socket(configuration.getString(CK_ZABBIX_HOST), configuration.getInt(CK_ZABBIX_PORT));
+
+        if (!socket.isConnected()) {
+            throw new Exception("failed to connect to zabbix server");
+        }
+        LOG.info("connection to zabbix server has been established");
+
+        buildMessageAndSend(message, socket);
+        readAndHandleResponse(socket);
+    }
+
+    private void buildMessageAndSend(Message message, Socket socket) throws IOException {
+        String str = buildMessage(message);
+        ByteBuffer buffer = getMessageByteBuffer(str);
+
+        OutputStream outputStream = socket.getOutputStream();
+        outputStream.write(buffer.array());
+    }
+
+    private void readAndHandleResponse(Socket socket) throws IOException {
+        String response = new String();
+        InputStream inputStream = socket.getInputStream();
+        while (isRunning()) {
```

Comment:
"while" clause is used because zabbix sends parts of the message which are aggregated in "response" variable. When message is sent, socket closes, as stream.read() returns "-1" value